monitoring/advanced-tracing.mdx

```diff
@@ -1,28 +1,29 @@
 ---
 title: Advanced tracing features
 sidebarTitle: Advanced tracing features
-description: Tracing additional data such as trace name, custom log, session ID, etc
+description: Tracing additional data such as trace name, custom log, session ID, etc.
 ---
 
 This page goes over some advanced tracing features.
 
 ## Adding a trace name
-
+{/* TODO: this section baad. I don't know how to fix it as I don't know what intention is here, what we actually want to show or explain */}
```

Comment:
I think this is a valid section, because if you don't add the name parameter you'll end up with a trace list that looks like this: 
<img width="173" alt="image" src="https://github.com/baserun-ai/docs/assets/1614514/8ce55d97-ee0c-44ea-a156-1abb029229c3">

I would change the description above the code examples to match both Python and TS code though, and maybe change the name parameter value to match what's on the screenshot below the code samples.